srcdoc/configure/access_control.mt

```diff
@@ -241,4 +241,33 @@ paths:
 EOT
 ?>
 
+<h2 id="how-to" class="section-head">How-To</h2>
+
+<h3 id="cidr">IP Address Matching Against CIDR list</h3>
+
+<p>
+You can match an IP address against predefined CIDR list using a script named <a href="">trie_addr.rb</a>.
+</p>
+<p>
+Below is an example.
+</p>
+
+<?= $ctx->{example}->('CIDR-based Address Matching Example', <<'EOT');
+paths:
+  "/":
+    mruby.handler: |
+      require "trie_addr.rb"
+      trie = TrieAddr.new.add(["192.168.0.0/16", "172.16.0.0/12"])
+      acl {
+        allow { trie.match?(addr) }
+        deny
+      }
+    file.dir: /path/to/doc_root
+EOT
+?>
+
+<p>
+This library currently supports only IPv4 addresses. If you passed an invalid IPv4 address including IPv6 addresses, <code>TrieAddr#match?</code> returns <code>false</code>.
```

Comment:
"TrieAddr#match? returns false when it receives an invalid IPv4 address (including an IPv6 address) as an argument."?